src/ott/utils.py

```diff
@@ -107,19 +107,19 @@ def default_progress_fn(
 
       def progress_fn(status, *args):
         iteration, inner_iterations, total_iter, state = status
-        iteration = int(iteration)
+        iteration = int(iteration) + 1
```

Comment:
although I think this might be a bit more intuitive, we're losing in terms of consistency because `errors[0]` is typically the error after 1 iteration (and not `error[1]`), so maybe, even though slightly counterintuitive, we may want to default to 0 indexing (and indeed, in a way, `x_0` is the first iterate, `x_{-1}` the init!!). That being said this is not a hard opinion, and i can imagine sticking with this convention. any opinion @michalk8  ?

Reply:
Agree, would still go for 0-based indexing.

Reply:
I understand, but I think users expect that accumulated values in an array will be 0-based, but that progress is 1-based.

Ordinarily, `1/2000` states that 1 iteration is finished and `2000/2000` means the last iteration is finished. This is the kind of convention found in training loops such as the [flax MNIST](https://colab.research.google.com/github/google/flax/blob/main/examples/mnist/mnist.ipynb) one: see, I picked an example from the jax ecosystem ;)

With 0-based indexing, the reported progress seems a bit awkward:

```
0/2000
1/2000
....
1999/2000
done
```

An alternative would be to report progress in %.

Which one do you prefer?
- 0-based
- 1-based
- percent

Reply:
We could also print both:

```
1/2000    (iter #0)
...
2000/2000 (iter #1999)
```

Reply:
Thanks for the detailed answer, I'd then keep 1-based indexing for consistency.